src/utils/badgeColor.ts

```diff
@@ -0,0 +1,15 @@
+import { differenceInDays } from 'date-fns'
+import { STATUS_COLORS, DAYS_BEFORE_EXPIRATIONS } from './constants'
+
+export const getBadgeColor = (prescriptionEndDate: Date): string => {
```

Comment:
`expirationDateToColor()` maybe

Reply:
I've changed the function name with expirationDateToColor